### PR TITLE
Add payment config

### PR DESCRIPTION
**Pull request by Endorf**
## Description of changes
- Added payment config

### CODE REVIEW

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/payment/OrderDetails.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+internal data class OrderDetails(
+    val price: Int,
+    val currency: String,
+    val raw: Raw
+) : Parcelable {
+
+    @kotlinx.parcelize.Parcelize
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/config/payment/OrderDetails.kt:13]: @Endorf Looks like u use different Parcelable annotations

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/core/CompositeCommand.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package com.verygoodsecurity.vgscheckout.networking.command.core
+
+import android.content.Context
+import com.verygoodsecurity.vgscheckout.exception.VGSCheckoutException
+import com.verygoodsecurity.vgscheckout.networking.command.GetOrderDetails
+import com.verygoodsecurity.vgscheckout.networking.command.GetSavedCardsCommand
+
+internal class CompositeCommand(
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/core/CompositeCommand.kt:8]: @Endorf Do we use this class anywhere?

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/core/CompositeCommand.kt [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    private var processedCommandsCounter = 0
+
+    fun execute(onResult: (Result) -> Unit) {
+        execute(Params(commands), onResult)
+    }
+
+    override fun run(params: Params, onResult: (Result) -> Unit) {
+        params.commands.forEach { command ->
+            when (command) {
+                is GetSavedCardsCommand -> command.execute {
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/core/CompositeCommand.kt:22]: @Endorf I think in this case, we can create not`CompositeCommand`, but for example `GetPaymentDetailsCommand` etc., and make two requests and only then return a result. I think  `CompositeCommand` name is too general for this case.

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/core/CompositeCommand.kt (hunk 2 above)
Comment (Endorf) [reply vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/core/CompositeCommand.kt:8]: Yes. In a Payment config. I just updated code.

File: vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/AddCardCommand.kt [hunk 4]
```diff
@@ -13,8 +13,11 @@ import com.verygoodsecurity.vgscheckout.util.extension.toAddCardResult
 import com.verygoodsecurity.vgscheckout.util.extension.toCollectMergePolicy
 import com.verygoodsecurity.vgscheckout.util.extension.toInternal
 
-internal class AddCardCommand constructor(context: Context) :
-    Command<AddCardCommand.Params, AddCardCommand.Result>(context) {
+internal class AddCardCommand constructor(
+    context: Context,
+    params: Params,
+) :
```
Comment (DmytroDm) [inline vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/AddCardCommand.kt:19]: @Endorf Can be one line?

Comment (Endorf) [reply vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/AddCardCommand.kt:19]: Sure. It is better to read in my case.

**Review by DmytroDm — APPROVED** [review]

**Pull request merged**